remote_cluster.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+source common.sh
+
+export REMOTE_CLUSTER_NAME=${REMOTE_CLUSTER_NAME:-remcluster}
+export REMOTE_CLUSTER_NETWORK_NAME=${REMOTE_CLUSTER_NETWORK_NAME:-${REMOTE_CLUSTER_NAME}net}
+export REMOTE_CLUSTER_SUBNET_V4=${EXTERNAL_SUBNET_V4:-"192.168.133.0/24"}
+export REMOTE_CLUSTER_SUBNET_V6=${EXTERNAL_SUBNET_V6:-"fd2e:6f44:5dd8:c960::/120"}
+export BMC_DRIVER=redfish-virtualmedia
+
+PLAYBOOK=${VM_SETUP_PATH}/setup-playbook.yml
+if [ "$1" == "teardown" ]  || [ "$1" == "clean" ]; then
+        PLAYBOOK=${VM_SETUP_PATH}/teardown-playbook.yml
+elif [ "$1" != "create" ]; then
+        echo "Usage: $0 <create | teardown>. Default is 'create'"
+        echo "Unknown command to $0. Exiting..."
+	exit 1
+fi
+
+ansible-playbook \
+    -e @remote_vm_setup_vars.yml \
+    -e "ironic_prefix=${REMOTE_CLUSTER_NAME}_" \
+    -e "cluster_name=${REMOTE_CLUSTER_NAME}" \
+    -e "remote_cluster_network_name=${REMOTE_CLUSTER_NETWORK_NAME}" \
+    -e "working_dir=$WORKING_DIR" \
+    -e "extradisks=$VM_EXTRADISKS" \
+    -e "libvirt_firmware=uefi" \
+    -e "virthost=$HOSTNAME" \
+    -e "vm_platform=$NODES_PLATFORM" \
+    -e "provisioning_url_host=$PROVISIONING_URL_HOST" \
+    -e "nodes_file=$NODES_FILE" \
+    -e "virtualbmc_base_port=$VBMC_BASE_PORT" \
+    -e "master_hostname_format=$MASTER_HOSTNAME_FORMAT" \
+    -e "worker_hostname_format=$WORKER_HOSTNAME_FORMAT" \
+    -i ${VM_SETUP_PATH}/inventory.ini \
+    -b -vvv ${PLAYBOOK}
+
```

Comment:
I think we'll want to run `generate_ocp_host_manifest` here so we generate the BMH manifests to be consumed on the hub cluster.

I also wonder if the script should be e.g `create_remote_nodes.sh` and make the cluster/network names default to those of the hub cluster, then this same script could be used for remote-worker-node testing?

Reply:
Thanks for the tip. I knew I was missing the manifests but couldn't quite figure out where I was going wrong.